1657395: Add descriptions for SEO (2 of 3) (#18320)
dotnet/docs: docs/framework/windows-workflow-foundation/overview.md

---
title: "Windows Workflow Overview"

> description: This article describes Workflow Foundation workflows, which are sets of units called activities stored as models that describes a real-world processes.

Reviewer: ```suggestion
description: This article describes Workflow Foundation workflows, which are models that describe real-world processes.
```
